.github/workflows/stale_bot.yml

```diff
@@ -0,0 +1,34 @@
+name: Stale Bot
+
+on:
+  schedule:
+#            ┌───────── minute (0 - 59)
+#            │ ┌───────── hour (0 - 23)
+#            │ │ ┌───────── day of the month (1 - 31)
+#            │ │ │ ┌───────── month (1 - 12 or JAN-DEC)
+#            │ │ │ │ ┌───────── day of the week (0 - 6 or SUN-SAT)
+    - cron: '0 1 * * 1' # Every Monday at 1am UTC
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/stale@v3
+      with:
+        repo-token: ${{ secrets.GITHUB_TOKEN }}
+        operations-per-run: 50
+        days-before-issue-stale: 99999999 # Never mark issues as stale
+        days-before-pr-stale: 60
+        days-before-close: 30
+        stale-pr-label: 'Stale'
+        stale-pr-message: >-
+          This pull request has been automatically marked as stale because it has not
+          had any activity for the past five months. It will be closed if no further
```

Comment:
```suggestion
          had any activity for the past month. It will be closed if no further
```

Reply:
I think something like 2 or 3 months would be better

Reply:
The PR as is, is currently, label and post after 2 months close after a further month.

Reply:
Since I'm out voted, I will remove this.